CMakeLists.txt

```diff
@@ -576,12 +576,8 @@ if(WIN32 OR WINCE)
 	add_plugin(support/win32 "Windows detected" TRUE)
 
 	SET(CMAKE_RC_COMPILE_OBJECT "<CMAKE_RC_COMPILER> -O coff -I ${CMAKE_CURRENT_SOURCE_DIR}/navit/gui/win32/resources <DEFINES> -i <SOURCE> -o <OBJECT>")
-	set_with_reason(support/ezxml "Windows detected" TRUE)
-	set_with_reason(speech/espeak "Windows detected" TRUE)
 	#speech/espeak does use windows threads, and therefore needs only parts of
```

Comment:
Please move comment as well, as this describes why INTERNAL_ESPEAK_COMPLETE is set to false.

Reply:
Moved the comment as requested. Please check again.